homestead.root/usr/share/clearwater/infrastructure/scripts/create-homestead-nginx-config

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# @file create-homestead-nginx-config
+#
+# Project Clearwater - IMS in the Cloud
+# Copyright (C) 2016 Metaswitch Networks Ltd
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or (at your
+# option) any later version, along with the "Special Exception" for use of
+# the program along with SSL, set forth below. This program is distributed
+# in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details. You should have received a copy of the GNU General Public
+# License along with this program.  If not, see
+# <http://www.gnu.org/licenses/>.
+#
+# The author can be reached by email at clearwater@metaswitch.com or by
+# post at Metaswitch Networks Ltd, 100 Church St, Enfield EN2 6BQ, UK
+#
+# Special Exception
+# Metaswitch Networks Ltd  grants you permission to copy, modify,
+# propagate, and distribute a work formed by combining OpenSSL with The
+# Software, or a work derivative of such a combination, even if such
+# copying, modification, propagation, or distribution would otherwise
+# violate the terms of the GPL. You must comply with the GPL in all
+# respects for all of the code used other than OpenSSL.
+# "OpenSSL" means OpenSSL toolkit software distributed by the OpenSSL
+# Project and licensed under the OpenSSL Licenses, or a work based on such
+# software and licensed under the OpenSSL Licenses.
+# "OpenSSL Licenses" means the OpenSSL License and Original SSLeay License
+# under which the OpenSSL Project distributes the OpenSSL toolkit software,
+# as those licenses appear in the file LICENSE-OPENSSL.
+
+# This file creates an nginx config file for homestead.
+
+. /etc/clearwater/config
+
+split_domain_and_port()
+{
+  echo $1 | perl -ne '$_ =~ /(.+):([0-9]+)$/ && print "$1 $2\n"'
+}
+
+if [ -n "$hs_mgmt_hostname" ]
+then
+  site_file=/etc/nginx/sites-available/homestead
+  enabled_file=/etc/nginx/sites-enabled/homestead
+  temp_file=$(mktemp homestead.nginx.XXXXXXXX)
+  server_name_and_port=( $( split_domain_and_port $hs_mgmt_hostname ) )
+
+  cat > $temp_file << EOF
+upstream http_homestead {
+        server unix:/tmp/homestead-http-mgmt-socket;
+
+        # The minimum number of idle connections to keep alive to the upstream.
+        keepalive 16;
+}
+
+server {
+        listen       [::]:8886 ipv6only=off;
```

Comment:
It's consistent though. If we want to change our products to respect the port specified in /etc/clearwater/shared_config I think we should do them all at once, otherwise it's confusing for us and for support.